content/docs/installation/continuous-deployment-and-gitops.md

```diff
@@ -112,3 +112,141 @@ Check the cert-manager logs for warnings and errors:
 ```bash
 kubectl logs -n cert-manager -l app.kubernetes.io/instance=cert-manager --prefix --all-containers
 ```
+
+
+## Using ArgoCD
+Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+### Pre-requisites
+Ensure the following are in place before proceeding:
+- A Kubernetes cluster
+- ArgoCD deployed on the Kubernetes cluster: [installation guide](https://argo-cd.readthedocs.io/en/stable/getting_started/)
+- Optional: A GitOps repository connected with ArgoCD: [setup guide](https://argo-cd.readthedocs.io/en/stable/user-guide/private-repositories/)
+
+### Setting up cert-manager
+1. Create an ArgoCD Application manifest file with the provided configuration to set up cert-manager.
```

Comment:
```suggestion
1. Create an [ArgoCD Application manifest file](link-to-reference-documentation) with the provided configuration to set up cert-manager.
```

It will be useful to link to the documentation so that the reader can learn about the fields of this custom resource.

Reply:
@wallrj this is not my code/fix BUT i'm currently affected by this.
I will pull this branch and make the suggested edits after my comments below are accepted or better suggestions are made.

Reply:
Thanks. I don't have permission to push to @shahkv95 's branch, so I've created my own.
 * https://github.com/cert-manager/website/pull/1447